.github/workflows/pr_labeler_validator.yaml

```
@@ -0,0 +1,152 @@
+        env:
+            IS_VALID_TRIGGER: ${{ github.event.action == 'opened' || github.event.changes.title.from }}
+        steps:
+            - name: Add labels to PR
+              id: labeler
+              uses: actions/labeler@v5
+              with:
+                  repo-token: ${{ secrets.GITHUB_TOKEN }}
+                  configuration-path: ".github/labeler.yaml"
+
+            - name: Check branch name format
+              run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)